featuremanagement/_featuremanagerbase.py

```diff
@@ -163,42 +164,46 @@ def _assign_variant(
         variant_name = None
         if not feature or not feature.variants or not feature.allocation:
             return
-        if feature.allocation.user and targeting_context.user_id:
-            for user_allocation in feature.allocation.user:
+
+        allocation = feature.allocation
```

Comment:
what's the difference between the variables `feature` and `feature_flag` ?

Reply:
I think the `feature_flag` value existed prior to the current usage of `evaluation_event`. I've cleaned it up.